Steepfile

```diff
@@ -92,5 +92,6 @@ target :elasticgraph_gems do
     # For more detail: https://github.com/soutaro/steep/wiki/Release-Note-1.5#better-flow-sensitive-typing-analysis
     config[::Steep::Diagnostic::Ruby::UnreachableBranch] = :hint
     config[::Steep::Diagnostic::Ruby::UnreachableValueBranch] = :hint
+    config[::Steep::Diagnostic::Ruby::UndeclaredMethodDefinition] = :information
```

Comment:
The new `UndeclaredMethodDefinition` diagnostics detects `def` syntaxes without type declaration. It detected some undeclared method definitions, so changed the diagnostics config to continue working for other type errors.